src/content/docs/en/guides/astro-db.mdx

```diff
@@ -156,6 +159,186 @@ export default async function() {
 
 Your development server will automatically restart your database whenever this file changes, regenerating your types and seeding your development data from `seed.ts`.
 
+## Connect a libSQL database for production
+
+Astro DB can connect to any libSQL server from any platform that exposes the libSQL remote protocol of the server, or can be self-hosted.
+
+To connect Astro DB to a libSQL database, set the following environment variables:
+
+- `ASTRO_DB_REMOTE_URL`: the connection URL to your libSQL server
+- `ASTRO_DB_APP_TOKEN`: the auth token to your libSQL server
+
+The way you get these values varies between services, but the next paragraph will show you how can you get started with [Turso](https://turso.tech), a fully managed database platform that uses libSQL.
```

Comment:
```suggestion
The way you get these values varies between services, but the next section will show you how can you get started with [Turso](https://turso.tech), a fully managed libSQL database platform.
```

Reply:
Should we point out that they created and maintain the libSQL server and protocol?

This sounds like any hosted Postgres platform, but it is more like recommending getting Redis from Redis Labs... It's literally their thing